### DISCUSSION TITLE
Certificate expired (Nightly Builds)

### DISCUSSION THREAD

**Discussion by Badkempachi** [glpi-project/glpi-agent#293] [General]
Hi, @g-bougard 
I'm sorry to bother you, I saw that you are answering questions more often here and I hope that you will be able to inform someone about the problem that has arisen.
Found that the nightly builds being signed are executed with an expired certificate.
There is no contact information on the pages with night builds, so I decided to publish the information here.
![cert](https://user-images.githubusercontent.com/97535165/207012025-a1c25527-e16f-4e38-87be-81de9ada7683.jpg)

**Comment by g-bougard** [comment]
Hi @Badkempachi 
indeed, that's the right place to alert us about such issue. We are in the process to renew the certificate as we won't be able to release any new official version for windows until the certificate is renewed.
